backends/arm/test/ops/test_depthwise_conv.py

```diff
@@ -130,11 +131,6 @@
 # Fails when enabling CompileSpec.set_quantize_io(True). MLETORCH-191.
 testsuite_u55.remove(("3x3_1x3x256x256_gp3_st1", dw_conv2d_3x3_1x3x256x256_gp3_st1))
 
-# Add failing test (set_quantize_io=True) temporarily to investigate
-testsuite_u55.append(
-    ("3x3_1x3x256x256_gp3_st1", dw_conv2d_3x3_1x3x256x256_gp3_st1, True)
-)
-
 
```

Comment:
This tests is causing problem in your CI dw_conv2d_3x3_1x3x256x256_gp3_st1 but works when we test it, lets remove it again (it was added back last week) for now and fix that in a separate patch after all is in.

Reply:
As a note the test running this was before this patch marked as @unittest.expectedFailure  but that line was removed below in this patch.